compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/SettingsBuilder.java

```diff
@@ -170,7 +174,12 @@ private Settings parseAsSettings() {
                         id = getStringOrDefaultFromTomlTableNode(repositoryNode, ID, "");
                         repositoryUsername = getStringOrDefaultFromTomlTableNode(repositoryNode, USERNAME, "");
                         repositoryPassword = getStringOrDefaultFromTomlTableNode(repositoryNode, ACCESS_TOKEN, "");
-                        repositories.add(Repository.from(id, url, repositoryUsername, repositoryPassword));
+                        String pathStr = getStringOrDefaultFromTomlTableNode(repositoryNode, "path", "");
```

Comment:
PATH?

Reply:
This is the only usage. So didn't think of adding a constant. But since all other fields are defined as constants, we can declare a constant for consistency.